### ISSUE TITLE
Add a new option to HOOK_WORKFLOW to ensure that the workflow ONLY start after the final forward model after the final update is finished

### ISSUE THREAD

**Issue by oddvarlia** [equinor/ert#8987]
Purpose: Be sure that an ERT workflow is run at the end after all iterations in ES-MDA is completed, but before ERT stops.
This is not possible now, and POST_SIMULATION will run the workflow after each iteration, not only after final iteration.

Solution: Implement a new option, e.g. called POST_EXPERIMENT for HOOK_WORKFLOW that start the workflow after the final ensemble after the last iteration is completed.

**Describe alternatives you've considered**
The workaround now is to run the ERT workflow manually from the workflow menu after the ERT run is completed, but I would like to have an option to run the ERT workflow automatic instead. Alternative solution is to test for ERT_ITERATION in the workflow script and only start the calculations when ERT_ITERATION is the final one and use HOOK_WORKFLOW with POST_SIMULATION option instead. In this case the ERT workflow will start after each iteration, but not do anything before the final iteration.

**Additional context**
Typical application: Use of the ERT workflow FIELD_STATISTICS recently implemented as a plugin for ERT in the subscript repo.

**Comment by sondreso** [comment]
Hi! Thank you for the feature request! I'll close this as a duplicate of https://github.com/equinor/ert/issues/7644, but this issue is linked to preserve the info!
